test/test_file.py

```diff
@@ -154,7 +154,16 @@ def test_preserve_times(self):
         # HERE an external access to the file is possible, modifying its access time
 
         # read times again and compare with original
-        st = os.stat(self.file.filename)
+        max_tries = 10
```

Comment:
Let's add a comment here explaining why this is necessary. This is so weird that we need to document it somehow.

Reply:
I would like to understand why it happens first.
Fixing the test is perhaps not enough...

Reply:
Agreed